src/components/Settings/SettingsNetwork/index.tsx

```diff
@@ -44,9 +38,19 @@ const messages = defineMessages('components.Settings.SettingsNetwork', {
     "Use ',' as a separator, and '*.' as a wildcard for subdomains",
   proxyBypassLocalAddresses: 'Bypass Proxy for Local Addresses',
   validationProxyPort: 'You must provide a valid port',
+  advancedNetworkSettings: 'Advanced Network Settings',
+  networkDisclaimer:
+    'Network parameters from your container/system should be used instead of this setting. See the {docs} for more information.',
```

Comment:
```suggestion
    'Network parameters from your container/system should be used instead of these settings. See the {docs} for more information.',
```